src/guiFormSpecMenu.cpp

```diff
@@ -1597,17 +1602,19 @@ void GUIFormSpecMenu::parseBackgroundColor(parserData* data,std::string element)
 	std::vector<std::string> parts = split(element,';');
 
 	if (((parts.size() == 1) || (parts.size() == 2)) ||
-		((parts.size() > 2) && (m_formspec_version > FORMSPEC_API_VERSION)))
-	{
-		parseColorString(parts[0],m_bgcolor,false);
+		((parts.size() > 2) && (m_formspec_version > FORMSPEC_API_VERSION))) {
+		parseColorString(parts[0], m_bgcolor, false);
```

Comment:
Depending on `parts[1]` (fullscreen), `m_fullscreen_bgcolor` should be set on this color instead of `m_bgcolor`.